src/server.js

```diff
@@ -22,26 +22,26 @@ app.use(express.static(path.resolve(__dirname, '../public')));
 
 // uploads a new script
 app.post('/script', scriptController.saveScript, scriptController.importScript, (req, res) => {
-  return res.status(200).json(res.locals.title);
+  return res.status(200).json({id: res.locals.id, title: res.locals.title});
 });
 
 // returns a list of all the script titles
 app.get('/scripts/title', scriptController.getScriptTitles, (req, res) => {
   return res.status(200).json(res.locals.scriptTitles);
 });
 
-// returns the script as a nested array
-app.get('/script/:title', scriptController.getScript, (req, res) => {
-  return res.status(200).json(res.locals.fullPlay);
+// returns the script text as a nested array
+app.get('/script/:script', scriptController.getScript, (req, res) => {
```

Comment:
Would it make sense to make this scriptId instead of just script? Especially with the contrast of having scriptText and the fact that on the front end referring to scripts is referencing an object with two properties. Not something that necessarily has to change, just not sure if that clears things up.

Reply:
I was dithering on that a bit since we currently have :actor instead of :actorId